### PR TITLE
adding server start to pytest hugectr script

### PR DESCRIPTION
**Pull request by jperez999**

### CODE REVIEW

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

File: tests/integration/hugectr/test_nvt_hugectr_training.py [hunk 1]
```diff
@@ -379,3 +381,20 @@ def _convert(data, slot_size_array):
...
+def test_server():
+    tri_cmd = [
+        "tritonserver",
+        "--model-repository=/model/models/",
+        "--backend-config=hugectr,test_model=/model/models/test_model/1/model.json",
+        "--backend-config=hugectr,supportlonglong=true",
+        "--model-control-mode=explicit",
+        "--load-model=test_model_ens",
+    ]
+    subprocess.Popen(tri_cmd)
```
Comment (benfred) [inline tests/integration/hugectr/test_nvt_hugectr_training.py:399]: What does this test?  I don't think this test will fail if the tritonserver fails to start. 

Also does this test leave a dangling tritonserver process running ? (like how is the process cleaned up)? 

also https://github.com/NVIDIA/NVTabular/pull/863/files#discussion_r646727169

**Pull request closed without merging**